test/Kestrel.FunctionalTests/AddressRegistrationTests.cs

```diff
@@ -98,7 +104,6 @@ public class AddressRegistrationTests : TestApplicationErrorLoggerLoggedTest
         [ConditionalTheory]
         [MemberData(nameof(IPEndPointRegistrationDataPort443))]
         [IPv6SupportedCondition]
-        [PortSupportedCondition(443)]
```

Comment:
The tests are failing because you missed a call to CanBindToEndpoint in RegisterIPEndPoint_Port443_Success.